deepctr_torch/layers/core.py

```diff
@@ -65,7 +71,7 @@ def forward(self, inputs):
             if self.use_bn:
                 fc = self.bn[i](fc)
 
-            fc = self.activation(fc)
+            fc = self.activation_layers[i](fc)
```

Comment:
Making the code less robust. You should make your code more robust by using `self.activation`